.travis.yml

```diff
@@ -11,6 +11,29 @@ env:
   - KARTOTHEK_ARROW_VERSION=0.12.1
   - KARTOTHEK_ARROW_VERSION=0.13.0
 
+
+matrix:
+  include:
+  - name: "Integration test for Numpy Pandas"
+    if: branch=numpan_nightly
```

Comment:
should be `if: branch-master` in the final version.

Reply:
added now